CheckDrive.Web/CheckDrive.Web/Controllers/DashboardController.cs

```diff
@@ -21,38 +21,46 @@ public DashboardController(IDashboardStore store, IAccountDataStore accountDataS
         public async Task<IActionResult> Index()
         {
             string token = HttpContext.Request.Cookies["tasty-cookies"];
+         
             var tokenHandler = new JwtSecurityTokenHandler();
             var jwtToken = tokenHandler.ReadToken(token) as JwtSecurityToken;
+            
             if (jwtToken == null)
             {
                 return RedirectToAction("Login", "Account");
             }
+            
             var accountId = jwtToken.Claims.First(claim => claim.Type == ClaimTypes.NameIdentifier).Value;
             int accountIds = Int32.Parse(accountId);
 
             TempData["UserName"] = _accountDataStore.GetAccountAsync(accountIds)
                 .Result.FirstName;
             TempData.Keep("UserName");
+            
             var dashboard = await _store.GetDashboard();
 
             if (dashboard is null)
             {
                 return BadRequest();
             }
+            
             SetViewBagProperties(dashboard);
 
             return View();
         }
         private void SetViewBagProperties(DashboardViewModel dashboard)
         {
             var summary = dashboard.Summary;
+            var Debts = Newtonsoft.Json.JsonConvert.SerializeObject(dashboard.Debts);
 
             ViewBag.MonthlyFuelConsumption = summary.MonthlyFuelConsumption.ToString("0.00");
             ViewBag.CarsCount = summary.CarsCount;
             ViewBag.DriversCount = summary.DriversCount;
 
             ViewBag.EmployeesCountByRole = dashboard.EmployeesCountByRoles;
             ViewBag.SplineChartData = dashboard.SplineCharts;
+            ViewBag.PetrolCount=dashboard.PetrolCounts;
```

Comment:
Other paths we use oil , it will be good if you rename petrol to oil

Reply:
Oil or Fuel